vtm-android-example/src/org/oscim/android/test/GdxActivity.java

```diff
@@ -65,6 +70,17 @@ public void createLayers() {
                     .httpFactory(new OkHttpEngine.OkHttpFactory())
                     .build();
             initDefaultLayers(ts, false, true, true);
+            resume();
```

Comment:
Better let life cycle calls to the system.

Reply:
resume wasn't called by system, kind of strange... :see_no_evil: